.github/workflows/release-notes.yaml

```
@@ -0,0 +1,76 @@
+name: Create Changelog and Release
+# Stick the format to v0.0.0 or V0.0.0 or 0.0.0
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)